### PR TITLE
Enable windows host metrics

### PR DESCRIPTION
**Pull request by lfitchett**
This re-enables host metrics for windows. They previously worked, but had to be disabled since they broke the windows arm-32 build. It also updates the sysinfo library

### CODE REVIEW

File: edgelet/edgelet-docker/src/runtime.rs [hunk 1]
```diff
@@ -636,85 +631,67 @@ impl ModuleRuntime for DockerModuleRuntime {
...
+            let mut info: libc::sysinfo = unsafe { mem::zeroed() };
+            let ret = unsafe { libc::sysinfo(&mut info) };
+            if ret == 0 {
+                info.uptime.try_into().unwrap_or_default()
+            } else {
+                0
+            }
+        };
+        #[cfg(windows)]
+        let uptime: u64 = unsafe { winapi::um::sysinfoapi::GetTickCount() }.into();
```
Comment (arsing) [inline edgelet/edgelet-docker/src/runtime.rs:645]: Use `GetTickCount64`. `GetTickCount` wraps after 50 days.

Comment (arsing) [inline edgelet/edgelet-docker/src/runtime.rs:645]: This is in milliseconds whereas the Linux one is in seconds. Should they not be consistent?

Comment (lfitchett) [reply edgelet/edgelet-docker/src/runtime.rs:645]: Changed

Comment (lfitchett) [reply edgelet/edgelet-docker/src/runtime.rs:645]: yes

**Pull request merged**